platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -136,7 +136,7 @@ class Viewer extends Component {
         timepointType: 'baseline',
         timepointId: 'TimepointId',
         studyInstanceUids: this.props.studyInstanceUids,
-        patientId: filter.patientId,
+        PatientID: filter.PatientID,
```

Comment:
How should we handle this?

Reply:
We call it with a filter which has a `PatientID` property right now. Its better that its consistent.